test/functional/ignore-list

```diff
@@ -13,3 +13,4 @@
 ^sh: .*/usr/bin/clr-boot-manager: No such file or directory$
 ^sh: .*/usr/bin/clr-boot-manager: not found$
 ^Possible filedescriptor leak: .*$
+^\.*$
```

Comment:
Oh sorry, I didn't notice the anchors... It's still not specific enough, since it will also match a blank line, and there is already an ignore entry for that. Maybe `^\.+$` instead?